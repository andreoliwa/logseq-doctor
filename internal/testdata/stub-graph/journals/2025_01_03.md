- My work
- Content after insertion point
