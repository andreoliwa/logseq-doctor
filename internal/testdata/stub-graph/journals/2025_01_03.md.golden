- My work
	- DOING 21:12 Some task
	  collapsed:: true
	  :LOGBOOK:
	  CLOCK: [2025-08-27 Wed 21:12:50]
	  :END:
- Content after insertion point